dev/cli

```diff
@@ -46,6 +47,24 @@ function enable_oidc() {
   env_args="$env_args $keycloak_env_args"
 }
 
+function enable_azure() {
+
+  . ../ci/authn-azure/check_dependencies.sh
+
+  echo "Setting Azure details as env variables"
```

Comment:
Let's keep the echos uniform across all  enable_authenticators

For oidc it is `echo "Extracting keycloak variables & setting as env variables"`

For azure it is `echo "Setting Azure details as env variables"`